Loop/Views/SettingsViewModel.swift

```diff
@@ -62,9 +64,11 @@ public class SettingsViewModel: ObservableObject {
                 therapySettings: TherapySettings,
                 supportedInsulinModelSettings: SupportedInsulinModelSettings,
                 pumpSupportedIncrements: PumpSupportedIncrements?,
+                pumpSyncSchedule: PumpManager.SyncSchedule?,
```

Comment:
What is the pump sync schedule used for?

Reply:
It is something the `BasalRateScheduleEditor` needs.  I'm *guessing* it syncs the schedule to the pump, but TBH I'm not sure why it isn't just part of `onSave`.  Right now I'm really just wiring things up, and avoiding changing behavior as much as possible.

Reply:
Syncs the schedule to the pump, yes. Separate from `onSave` since the 'loading' state and errors occurred during syncing is handled in the editor, which means the client knows that when `onSave` is invoked, the syncing process has completed successfully.